[Android] `ALooper_pollAll` is deprecated in r27

**Describe the bug**

The examples use a deprecated android api `ALooper_pollAll` and with r27 it is removed with a compilation error.
The default github runners are now on that version by default.


**To Reproduce**
Compile bgfx for android using r27 ndk or newer

**Expected behavior**
Compilation finishes without error.

**Screenshots**

```
/home/runner/work/bgfx.cmake/bgfx.cmake/bgfx/examples/common/entry/entry_android.cpp:194:22: error: 'ALooper_pollAll' is unavailable: obsoleted in Android 1 - ALooper_pollAll may ignore wakes. Use ALooper_pollOnce instead. See The API documentation for more information
  194 |                                 /*int32_t id =*/ ALooper_pollAll(-1, NULL, &num, (void**)&source);
      |                                                  ^
/usr/local/lib/android/sdk/ndk/27.0.12077973/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include/android/looper.h:228:5: note: 'ALooper_pollAll' has been explicitly marked unavailable here
  228 | int ALooper_pollAll(int timeoutMillis, int* outFd, int* outEvents, void** outData)
      |     ^
```

**Additional context**
https://github.com/bkaradzic/bgfx.cmake/actions/runs/10201945348/job/28225018352?pr=239
https://github.com/android/ndk/discussions/2020

--- a/examples/common/entry/entry_android.cpp
+++ b/examples/common/entry/entry_android.cpp
@@ -189,9 +189,10 @@ namespace entry
 
 			while (0 == m_app->destroyRequested)
 			{
-				int32_t num;
 				android_poll_source* source;
-				/*int32_t id =*/ ALooper_pollAll(-1, NULL, &num, (void**)&source);
+				int32_t result = ALooper_pollOnce(-1, NULL, NULL, reinterpret_cast<void**>(&source));
+
+				BX_ASSERT(ALOOPER_POLL_ERROR != result, "ALooper_pollOnce returned an error.");
 
 				if (NULL != source)
 				{

